src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/IntellisenseQuickInfoBuilder.cs

```diff
@@ -41,21 +42,48 @@ internal static class IntellisenseQuickInfoBuilder
                 firstLineElements.Add(new ImageElement(warningGlyph.GetImageId()));
             }
 
+            var elements = new List<object>();
             var descSection = quickInfoItem.Sections.FirstOrDefault(s => s.Kind == QuickInfoSectionKinds.Description);
             if (descSection != null)
             {
-                firstLineElements.Add(BuildClassifiedTextElement(descSection));
+                var isFirstElement = true;
+                foreach (var element in BuildClassifiedTextElements(descSection))
```

Comment:
this code could use same pattern as the one before? the documentationCommentSection?

Reply:
The sections now use similar code patterns. It's not perfect, but the test coverage should be much better and combined with the new documentation it lower our chances of a regression.